### PR TITLE
Manager Artifacts - Eng.QAIntegrationTesting - 26/09/2019

### PR DESCRIPTION
**Pull request by inesOfAurea**

### CODE REVIEW

File: Managers/Ines.Guisado-coaching-Eng.QAIntegrationTesting.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+teamroom: Eng.QAIntegrationTesting
+direct reports: 4
+ICs:
+    - IC:
+        IC name: David Gunadi 
+        date: 09/26/2019
+        unit: https://jira.devfactory.com/browse/SBM-55857
+        coaching: Coach on how to find previous automation
+        performance assessment: highPerformer
+        continue coaching: no
```
Comment (alexey-aurea) [inline Managers/Ines.Guisado-coaching-Eng.QAIntegrationTesting.yaml]: please use true | false as per the schema file

File: Managers/Ines.Guisado-process-improvement-Eng.QAIntegrationTesting.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+manager: Ines Guisado
+teamroom: Eng.QAIntegrationTesting
+units:
+    - unit: https://jira.devfactory.com/browse/PROLOGICCID-5444
+recommendation:
+    eliminate waste: yes
+    improve quality bar: yes
```
Comment (alexey-aurea) [inline Managers/Ines.Guisado-process-improvement-Eng.QAIntegrationTesting.yaml]: please use true | false as per the schema file

File: Managers/Ines.Guisado-process-improvement-Eng.QAIntegrationTesting.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+manager: Ines Guisado
+teamroom: Eng.QAIntegrationTesting
+units:
+    - unit: https://jira.devfactory.com/browse/PROLOGICCID-5444
+recommendation:
+    eliminate waste: yes
```
Comment (alexey-aurea) [inline Managers/Ines.Guisado-process-improvement-Eng.QAIntegrationTesting.yaml]: please use true | false as per the schema file

File: Managers/Ines.Guisado-automation-Eng.QAIntegrationTesting.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+manager: Ines Guisado
+teamroom: Eng.QAIntegrationTesting
+units:
+    - unit: https://jira.devfactory.com/browse/QAP-8578
+automation:
+    product agnostic: true
+    product specific:
+        - product name:
```
[the remark this answers is not in the document]
Comment (inesOfAurea) [reply Managers/Ines.Guisado-automation-Eng.QAIntegrationTesting.yaml]: Done

File: Managers/Ines.Guisado-process-improvement-Eng.QAIntegrationTesting.yaml (hunk 3 above)
Comment (inesOfAurea) [reply Managers/Ines.Guisado-process-improvement-Eng.QAIntegrationTesting.yaml]: Done.

File: Managers/Ines.Guisado-process-improvement-Eng.QAIntegrationTesting.yaml (hunk 2 above)
Comment (inesOfAurea) [reply Managers/Ines.Guisado-process-improvement-Eng.QAIntegrationTesting.yaml]: Done

File: Managers/Ines.Guisado-coaching-Eng.QAIntegrationTesting.yaml (hunk 1 above)
Comment (inesOfAurea) [reply Managers/Ines.Guisado-coaching-Eng.QAIntegrationTesting.yaml]: Done

**Pull request merged**